test/tests/performance/pbs_preemptperformance.py

```diff
@@ -480,6 +480,69 @@ def test_preemption_basic(self):
             ncpus *= 3
             S_jobs += ncpus
 
+    @timeout(3600)
+    def test_preemption_with_soft_limits(self):
+        """
+        Measure the time scheduler takes to preempt when the high priority
+        job hits soft limits under a considerable amount of workload.
+        """
+        a = {'resources_available.ncpus': 4,
+             'resources_available.mem': '6400mb'}
+        self.server.create_vnodes('vn', a, 500, self.mom, usenatvnode=False)
+        p = "express_queue, normal_jobs, server_softlimits, queue_softlimits"
+        a = {'preempt_prio': p}
+        self.server.manager(MGR_CMD_SET, SCHED, a, runas=ROOT_USER)
```

Comment:
The user running the test is a manager.  You don't need to run this as ROOT_USER.